core/domain/exp_domain_test.py

```diff
@@ -5232,14 +6936,10 @@ def test_load_from_v26_textangular(self):
         mock_get_filename_with_dimensions_context = self.swap(
             html_validation_service, 'get_filename_with_dimensions',
             mock_get_filename_with_dimensions)
-        mock_validate_rte_format_for_v26_context = self.swap(
-            html_validation_service, 'validate_rte_format',
```

Comment:
Why we have removed this?

Reply:
The test does not depend on `validate_rte_format` and hence this function(`mock_validate_rte_format_for_v26`) was untested

Reply:
@lilithxxx, Thanks for the clarification. :)

@bansalnitish, we were expecting the exploration migration/validation to be dependent on this method? have we removed `validate_rte_format` from the codebase unexpectedly? (Asking you just to make sure that we can remove these lines*)

Reply:
Hi @DubeySandeep @lilithxxx, sorry for the delay in response. This function was added by @ankita240796 when we added validation to the html fields but since the validation was removed due to exploration migration failure, this function does not serve any purpose for the testing. So, it is fine to remove it. (We have not removed `validate_rte_format` from the codebase but we have removed the call to this function while saving html for an exploration)